### DISCUSSION TITLE
Change from Redis to Valkey in 3.0.0

### DISCUSSION THREAD

**Discussion by szeestraten-nina** [netbox-community/netbox-docker#1324] [Getting Help]
Hey there! I see Redis was swapped out for Valkey in 3.0.0 #1203.
We have a running setup based on Netbox v4.0.8 and netbox-docker 2.9.1 with Redis which we would like to update to 3.0.1.
Is there anything special that needs to be done for such an upgrade when swapping out Redis for Valkey?

**Comment by NurAabideen** [comment]
Did you upgrade to ValKey? anything special you had to do the config?
